src/Azure.php

```diff
@@ -31,12 +33,12 @@ public function handle($request, Closure $next)
 
         if (env("APP_ENV") === "testing")
```

Comment:
`config('app.env')` seems prefered to `env()`. Mitigates differences if config files are cached.